certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -217,6 +217,408 @@
           }
         ]
       }
+    },
+    "/GetServiceAssignmentsDispatcher": {
```

Comment:
The array contains duplicate values, values must be unique. ValuePath: /paths/~1GetServiceAssignmentsDispatcher/post/parameters/0/schema/required/5